requirements.txt

```diff
@@ -16,29 +17,28 @@ gunicorn
 hive-thrift-py
 ipython
 jinja2
-librabbitmq
 markdown
-mysql-python
 nose
+nose-exclude
 pandas
 pygments
 pyhive
 pydruid
 PySmbClient
 python-dateutil
-#pyhs2
+# pyhs2 -> not compatible with Python 3 because of sasl
 redis
 requests
 setproctitle
 statsd
-#snakebite -> not compatible with Python 3
+# snakebite -> not compatible with Python 3
 sphinx
 sphinx-argparse
 sphinx_rtd_theme
 Sphinx-PyPI-upload
 sqlalchemy
 thrift
 jaydebeapi
-#pymssql
+mysqlclient
```

Comment:
this seems to introduce an install-time dependency on mysql as mysqlclient looks for mysql_config